shared/chat/conversation/messages/message-popup/attachment.tsx

```diff
@@ -79,6 +92,8 @@ const PopAttach = (ownProps: OwnProps) => {
     : []
   const itemMedia = [{icon: 'iconfont-camera', onClick: onAllMedia, title: 'All media'}] as const
 
+  const itemJump = [{icon: 'iconfont-search', onClick: onJump, title: 'Jump to message'}] as const
```

Comment:
i debated hiding this if you click from the thread but its benign and isn't likely worth the plumbing and its nicer to not have this menu change subtly